src/test/pmempool_sync_remote/TEST0

```diff
@@ -39,10 +39,10 @@
 
 require_test_type medium
 
-. common.sh
-
 setup
 
+. common.sh
```

Comment:
It's not that simple. common.sh includes some requirements, which should be executed before "setup".